runtime/bcverify/staticverify.c

```diff
@@ -1421,19 +1429,18 @@ checkStackMapEntries (J9CfrClassFile* classfile, J9CfrAttributeCode * code, U_8
 			}
 		}
 
-		/* locals:
-		 * 		It is an error if, for any index i, locals[i] represents a local variable whose index is greater than the maximum 
-		 * 		number of local variables for the method. 
-		 * stack:
-		 * 		It is an error if, for any index i, stack[i] represents a stack entry whose index is greater than the maximum operand stack size for the method.
-		 */
-		slotTypeCounter += 1;
-		if ((CFR_STACKMAP_TYPE_DOUBLE == entryType) || (CFR_STACKMAP_TYPE_LONG == entryType)) {
+		/* A value of type long or double must occupy two consecutive local variables. Ensure that if there is is a long or double entry in 
+		 * an append frame maxLocals reflects the correct number of slots. An incorrect maxLocals value in all other cases will be handled 
+		 * in bcverify.c */
```

Comment:
Why isn't the append frame handled there if all other frame types are?

Reply:
This is to match error types thrown by the RI.